### ISSUE TITLE
Carousel keyboard navigation

### ISSUE THREAD

**Issue by gmerabishvili** [primefaces/primeng#8366]
Hello, Keyboard Navigation doesn't work on Carousel. Can you fix that please?

**Comment by cagataycivici** [comment]
Unable to replicate with PrimeNG 9, if the issue persists please create a new ticket with a test case reproducing the issue e.g. stackblitz or a github repo and it will be reviewed by our team once again.

**Comment by gmerabishvili** [comment]
Thank you very much! Appreciate!

**Comment by itworksafisher** [comment]
How does a keyboard user easily get to the next chevron button? They can go backwards easily since the previous chevron is the first tabbable element. Otherwise they have to tab through all the carousel content, which breaks down the experience for them in a carousel which shows and slides multiple items at once. It also makes it difficult for them to skip the carousel since all content must be tabbed through to do so. The experience would be improved as a single tab stop, where left and right arrow would focus and activate the chevrons, and perhaps down and up arrow would bring them in out of the carousel content. Tabbing would move them away from the carousel to the next tabbable element on the page.
